sklearn/datasets/base.py

```diff
@@ -144,11 +144,10 @@ def load_files(container_path, description=None, categories=None,
         contains characters not of the given `encoding`. Passed as keyword
         argument 'errors' to bytes.decode.
 
-    random_state : int, RandomState instance or None, optional (default=0)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default=0)
+        Determines random number generation for dataset shuffling. Pass an int
+        for reproducible output across multiple function calls.
+        See :term:`random_state <Glossary>`.
```

Comment:
This should be,
```
:term:`Glossary <random_state>`.
```
as this link doesn't currently [render correctly](https://20327-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.datasets.fetch_kddcup99.html)

Reply:
Good catch. Thanks @rth.